app/helpers/application_helper.rb

```diff
@@ -2,4 +2,8 @@ module ApplicationHelper
   def rating_options(selected=nil)
     options_for_select((1..5).map {|num| [pluralize(num,"star"), num.to_f]}, selected)
   end
+
+  def random_users
```

Comment:
This is more like a class method to User class. Data manipulation - modern concern, not a helper presentation concern.